todos-app/src/App/ListFooter/FilterList/FilterList.tsx

```diff
@@ -1,7 +1,55 @@
-import React from "react";
+import { FilterType } from "../../models/Filters";
+import "../../../App.css";
 
-const FilterList: React.FC = () => {
-  return <div></div>;
+type Props = {
+  filterTodosHandler(filterValue: string): void;
+};
+
+const FilterList = ({ filterTodosHandler }: Props) => {
+  const setClassName = (flag: FilterType): void => {
+    const selectedAll = document.querySelectorAll("li > a");
+    for (let element of selectedAll) {
+      if (element.textContent === flag) {
+        element.className = "selected";
```

Comment:
This works but it's not exactly the idiomatic React way, how can you track which filter is selected so that you can dynamically assign a class name in the JSX?